### PR TITLE
(DOCS) Update docs and changelog for `alpha.3`

### PR DESCRIPTION
**Pull request by michaeltlombardi**
# PR Summary

This change updates the reference documentation and the changelog to reflect the changes that were included in the `alpha.3` release:

- #197
- #198
- #199

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,10 +20,74 @@ This section includes a summary of user-facing changes since the last release. F
...
+
+  <details><summary>Related work items</summary>
+
+  - PRs: [#198][#198]
+
+  </details>
+
+- The `input` property for the [get][18], [set][19], and [test][20] commands in a resource manifest
+  no longer has a default value. Instead, when a resource doesn't define `input` for a command, DSC
+  doesn't send any input to the resource for that command.
```
Comment (michaeltlombardi) [inline CHANGELOG.md:83]: @SteveL-MSFT - this was my understanding from the PR, but I'm not _entirely_ sure - does this change actually only affect `get`? If so, I can update this PR and #203 to reflect that.

File: CHANGELOG.md [hunk 2]
```diff
@@ -219,19 +283,32 @@ For the full list of changes in this release, see the [diff on GitHub][compare-v
...
+[#176]: https://github.com/PowerShell/DSC/176
+[#177]: https://github.com/PowerShell/DSC/177
+[#181]: https://github.com/PowerShell/DSC/181
+[#182]: https://github.com/PowerShell/DSC/182
+[#197]: https://github.com/PowerShell/DSC/197
+[#198]: https://github.com/PowerShell/DSC/198
+[#199]: https://github.com/PowerShell/DSC/199
+[#45]:  https://github.com/PowerShell/DSC/45
+[#73]:  https://github.com/PowerShell/DSC/73
+[#98]:  https://github.com/PowerShell/DSC/98
```
Comment (sdwheeler) [inline CHANGELOG.md:314]: These links are all broken.

**Review by sdwheeler — CHANGES_REQUESTED** [review]
Fix the broken links.

**Review by sdwheeler — APPROVED** [review]

File: docs/reference/cli/resource/set.md [hunk 3]
```diff
@@ -31,17 +31,27 @@ This subcommand can only be invoked for command-based DSC Resources that define
...
+> [!IMPORTANT]
+> The `dsc resource set` command always invokes the `set` operation for the resource. Resources
+> may, but aren't required to, implement logic that pretests an instance for the `set` operation.
+>
+> This is different from how [dsc config set][02] works, where DSC always tests an instance, either
+> synthetically or by invoking the `test` operation for the resource, and only invokes `set` for an
+> instance if it's not in the desired state.
+>
+> If a resource doesn't implement pretest, it enforces the desired state regardless of the current
+> state when this command executes. If a resource does implement pretest, it only modifies the
```
Comment (SteveL-MSFT) [inline docs/reference/cli/resource/set.md:43]: Perhaps it'd be better to say something like: If a resource indicates it implements pretest, then the expectation is that the resource determines if any modification of configuration is required even if a `set` operation is requested.

**Review by SteveL-MSFT — COMMENTED** [review]
Added one suggestion

File: docs/reference/cli/resource/set.md (hunk 3 above)
Comment (michaeltlombardi) [reply docs/reference/cli/resource/set.md]: Reorganized the notice a little bit to move the manifest information above the impact and reworded it around expected behavior.

**Review by SteveL-MSFT — APPROVED** [review]

**Pull request merged**